plugins/samples/overwrite_header/tests.textpb

```diff
@@ -1,37 +1,47 @@
 # Expect no matches, so no overwrite.
 test {
-  name: "NoOverwrite"
-  benchmark: true
+  name: "NoOverwriteRequestHeader"
+  benchmark: false
   request_headers {
     input { header { key: "SomeRequestHeaderKey" value: "SomeHeaderValue" } }
```

Comment:
nit: If you want to add other headers into the mix (SomeRequestHeaderKey), you should show that they survive unchanged by testing in `result{}` that they still have the same value.